nixos/modules/services/networking/ntp/ntpd.nix

```diff
@@ -132,6 +132,7 @@ in
 
         preStart =
           ''
+            # shellcheck disable=SC2174
             mkdir -m 0755 -p ${stateDir}
```

Comment:
```suggestion
            mkdir -m 0755 ${stateDir} || true
```
what do you think about this?

Reply:
Perhaps this code can be removed completely by utilizing `StateDirectory` for the service. `stateDir` is hardcoded to be `/var/lib/ntp`

Reply:
Yeah, completely missed that!

Reply:
Good call.